ui/locales/en.json

```diff
@@ -2,6 +2,7 @@
   "AD_DURATION": "Ad duration",
   "AD_PROGRESS": "Ad [AD_ON] of [NUM_ADS]",
   "AD_TIME": "Ad: [AD_TIME]",
+  "AIRPLAY": "Cast...",
```

Comment:
Is "Cast" the terminology that is commonly used for AirPlay?

Reply:
It was either that or put the name of the technology itself (AirPlay), I have put Cast to maintain uniformity when sending to a 2nd screen like Chromecast, but if you have a better name I change it.

https://developer.apple.com/documentation/webkitjs/adding_an_airplay_button_to_your_safari_media_controls

Reply:
I think it'd be okay to just have the text of this be "AirPlay". It's an ARIA label, so it's supposed to tell a blind person what this button does. And I think that the button is specifically for interfacing with AirPlay-capable devices is useful context. There is, I suppose, the possibility that there might be a blind person who owns both a ChromeCast device and an Apple TV out there, somewhere; it'd be confusing if both buttons were just labeled "Cast...".

I was not the person who designed the UI, but I believe that the ARIA label for the Chromecast button is "Cast..." because that is what the Chromecast button in the Chrome browser's overflow menu is called.
Now, I don't have any AirPlay devices to test this myself, but according to screenshots on [a WikiHow article I found](https://www.wikihow.com/images/thumb/b/b9/Turn-On-AirPlay-Step-16-Version-3.jpg/aid6607080-v4-728px-Turn-On-AirPlay-Step-16-Version-3.jpg.webp), it looks like when AirPlay shows up in non-icon form on Apple menus, it usually is just expressed as "AirPlay". So I'm not 100% sure, but I think that the ellipses are not the critical part of these labels.

Reply:
I changed the name to "AirPlay". Thanks @theodab